packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -0,0 +1,497 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'package:flutter/cupertino.dart';
+
+const EdgeInsets _kHorizontalItemPadding =
+    const EdgeInsets.symmetric(horizontal: 16.0, vertical: 0.0);
+
+const Radius _kDefaultCornerRadius = const Radius.circular(3.0);
+
+/// An iOS-style segmented control bar
+///
+/// Displays the provided [children] widgets in a horizontal list.
+/// Exactly one [children] widget is always selected. By default,
+/// the leftmost widget is selected.
+///
+/// All widgets have the same width.
+///
+/// All widgets provided should have a Key value that can be used
+/// to identify the widget. Typically, these keys will be instances
+/// of the [ValueKey] class. This key is what will be returned in
+/// the inputted callback.
+///
+/// See also:
+///
+/// * <https://developer.apple.com/design/human-interface-guidelines/ios/controls/segmented-controls/>
+class SegmentedControl extends StatefulWidget {
+  /// Creates an iOS-style segmented control bar.
+  SegmentedControl({Key key, @required this.children, @required this.onValueChanged, this.value})
```

Comment:
Place a comma after this.value and re-run dartfmt. It'll format this the right way then. In general, long argument lists should have a trailing comma as a signal to dartfmt to wrap the arguments one-per-line.